src/computed/channel.js

```diff
@@ -67,6 +67,10 @@ const ComputedChannel = store => {
         .reduce((a, b) => a + b, 0);
       return toAmountLabel(sum, settings);
     },
+    get channelBalanceOnChainLabel() {
```

Comment:
Unit test augmentation? :) changes would go here: https://github.com/lightninglabs/lightning-app/blob/master/test/unit/computed/channel.spec.js#L57 and a few other places in the file.

Reply:
Fixed!